install.sh

```diff
@@ -7,43 +7,22 @@ echo "# This script installs the dependencies for emirge. #"
 echo "#####################################################"
 echo
 
-if [[ ! -f meshmode/setup.py ]]; then
-	echo "ERROR: incomplete git clone. Please run:"
-	echo "  git submodule init && git submodule update"
-	echo "to fetch emirge's submodules."
-	exit 1
-fi
+export MY_CONDA_DIR=$HOME/miniforge3
 
-myos=$(uname)
-[[ $myos == "Darwin" ]] && myos="MacOSX"
+./install-conda.sh
 
-myarch=$(uname -m)
+export PATH=$MY_CONDA_DIR/bin:$PATH
 
-if [[ ! -d ~/miniforge3 ]]; then
-    echo "Installing Miniforge"
-    wget -c --quiet https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$myos-$myarch.sh
-    bash Miniforge3-$myos-$myarch.sh -b
-fi
+echo "==== Create 'dgfem' conda environment"
+conda init
```

Comment:
Should `conda init` go here or in `install-conda.sh`? (Not really sure what makes more sense.)

Reply:
I think it's fine here.